meilisearch-core/src/settings.rs

```diff
@@ -98,17 +102,17 @@ pub enum RankingRule {
     Dsc(String),
 }
 
-impl ToString for RankingRule {
-    fn to_string(&self) -> String {
+impl std::fmt::Display for RankingRule {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match self {
-            RankingRule::Typo => "_typo".to_string(),
-            RankingRule::Words => "_words".to_string(),
-            RankingRule::Proximity => "_proximity".to_string(),
-            RankingRule::Attribute => "_attribute".to_string(),
-            RankingRule::WordsPosition => "_words_position".to_string(),
-            RankingRule::Exactness => "_exactness".to_string(),
-            RankingRule::Asc(field) => format!("asc({})", field),
-            RankingRule::Dsc(field) => format!("dsc({})", field),
+            RankingRule::Typo => write!(f, "typo"),
+            RankingRule::Words => write!(f, "words"),
+            RankingRule::Proximity => write!(f, "proximity"),
+            RankingRule::Attribute => write!(f, "attribute"),
+            RankingRule::WordsPosition => write!(f, "words_position"),
```

Comment:
I was wondering if we want to use snake or camel casing for the criterion names.
"wordsPosition" instead of "words_position".